### PR TITLE
OCM-3284 | fix: allow to choose load balancer type for sts clusters

### PR DESCRIPTION
**Pull request by gdbranco**
Related issue: https://issues.redhat.com/browse/OCM-3284

### CODE REVIEW

File: cmd/edit/ingress/cmd.go [hunk 1]
```diff
@@ -284,30 +284,22 @@ func run(cmd *cobra.Command, argv []string) {
...
 			os.Exit(1)
 		}
-		if ocm.IsSts(cluster) {
-			r.Reporter.Errorf("Updating Load Balancer Type is not supported for STS clusters")
-			os.Exit(1)
-		}
 		lbType = &args.lbType
-	} else {
-		if interactive.Enabled() {
-			if !ocm.IsSts(cluster) {
```
Comment (oriAdler) [inline cmd/edit/ingress/cmd.go:294]: With this check removed, do we need to check `!ocm.IsHyperShiftCluster(cluster)` before prompting the message?

Comment (gdbranco) [reply cmd/edit/ingress/cmd.go:294]: Oh true

Comment (gdbranco) [reply cmd/edit/ingress/cmd.go:294]: Fixed, thanks!

**Pull request merged**